platform/feature-api/api/manifest-parser/package.json

```diff
@@ -16,7 +16,6 @@
     "test": "jest"
   },
   "dependencies": {
-    "@pnpm/read-package-json": "^3.1.5",
```

Comment:
This needs to be eliminated from downstream package-lock.json's, but there's obviously no hurry to do so, it will happen naturally when these are changed.